recipes-bsp/rcw/rcw_git.bb

```diff
@@ -19,6 +19,7 @@ M="${@d.getVar('MACHINE').replace('-64b','').replace('-32b','').replace('-${SITE
 BOARD_TARGETS="${M}"
 BOARD_TARGETS_ls2088ardb="${M} ${M}_rev1.1"
 BOARD_TARGETS_ls1088ardb-pb="ls1088ardb"
+BOARD_TARGETS_lx2160ardb = "${M} ${M}_rev2"
```

Comment:
What does this support mean? Does is covers new development boards only? Is it for all boards?

Reply:
update commit message.

Reply:
@rehsack does this make sense to you? or any further explanation is needed on the commit log?

Reply:
It's like the two line above deal for LS2088A Reference Design Board - there're two revisions out, each. And it's left up to user which one is put into NOR-flash (or EMMC).
I would have preferred @ting-liu gives a better answer when to use rev1 or rev2, respectively - including what has to be changed when no NOR is available but eMMC only - but I think this could end in a translation of Effie Briest. It's likely to much for a commit to meta-freescale since NXP itself hides a lot of those documentation by obfuscation contests for documents.

Reply:
Rev 2 of LX2160A processor fixes some PCIe/qDMA/SATA/USB/FlexCAN issues. A new LX2160A Reference Design Board (https://www.nxp.com/part/LX2160A-RDB-B) ships with it. The LX2160A Chip Errata is under internal review and will be published on nxp.com later.

Dedicated rcw binaries in folder lx2160ardb_rev2 is for the Reference Design Board with Rev 2 processor. Old boards still use rcw binariess in folder lx2160ardb.